cmd/root.go

```diff
@@ -254,7 +255,7 @@ func createGithubClient(flag *flag.FlagSet, verifyFlags bool) (multigitter.Versi
 		Organizations: orgs,
 		Users:         users,
 		Repositories:  repoRefs,
-	}, mergeTypes)
+	}, mergeTypes, forkMode)
```

Comment:
Not sure what idiomatic Go is here but I generally try to avoid functions with more than three or four arguments as it becomes hard to remember what goes where. In Ruby I would turn this into keyword arguments; in Go I guess the analogue would be creating a small struct type with these arguments as fields?

Reply:
Agreed. It was already to many arguments before this addition. I will make this change, but after this PR 😃